src/main/java/racing/Car.java

```diff
@@ -0,0 +1,39 @@
+package racing;
+
+public class Car {
+
+    private static final int MOVE_THRESHOLD = 4;
+    private static final int MAX_NAME_LENGTH = 5;
+    private final String name;
```

Comment:
상수와 필드는 한 줄 띄어써서 구분하기 쉽게 해주세요. 🙏

```suggestion
    private static final int MAX_NAME_LENGTH = 5;
    
    private final String name;
```